P5/Source/Specs/accMat.xml

```diff
@@ -18,11 +18,7 @@ described, such as non-contemporaneous documents or fragments bound in
 with it at some earlier historical period.</desc>
   <desc versionDate="2007-12-20" xml:lang="ko">기술하고 있는 원고와 밀접히 관련된 중요한 부가적 자료를 세밀하게 기술한다. 예를 들어, 어떤 이전 시기에 원고와 함께 제본된 비동시대의 문서  또는 문서 일부.</desc>
   <desc versionDate="2007-05-02" xml:lang="zh-TW">包含可能與該手稿關係密切的重要附加資料之細節，例如非同時期的文件或是在早期與手稿裝訂一起的片段。</desc>
-  <desc versionDate="2008-04-05" xml:lang="ja">当該手書き資料と関連する情
-  報の詳細を記述するための要素．例えば，当該手書き資料が書かれた時代と
-  は異なる時期の文書や断片についてについての詳細を示す．例えば，当該手
-  書き資料が作成された以前に書かれた文書やその断片であったりするものに
-  ついて．
+  <desc versionDate="2008-04-05" xml:lang="ja">当該手書き資料と密接に関連していると考えられる重要な追加資料，例えば、時代を異にする文書，歴史的に早い時期に手書き資料と合わされた断片などの詳細を示す．
```

Comment:
`@versionDate` should be updated

Reply:
Dear Peter, thank you for your cheking. I've updated the date.